bundle.md

```diff
@@ -10,11 +10,11 @@ The definition of a bundle is only concerned with how a container, and its confi
 A Standard Container bundle contains all the information needed to load and run a container.
 This MUST include the following artifacts:
 
-1. `config.json`: contains configuration data.
+1. <a name="containerFormat01" />`config.json`: contains configuration data.
```

Comment:
These anchors look good to me, although I think we should stick to the lowercase, hyphenated form that GitHub and Pandoc are using already for headers.  In that case, the name here would be `container-format-01`.